src/Service/S3/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 1.0.1
```

Comment:
Isnt this a new feature of S3?

Reply:
Dont we also require to update the core version constraint in S3's composer.json

Reply:
switched to `1.1.0`.

The core is backward compatible, no need to bump the version.
People with s3:1.1.0 and core:1.0.0 will call the private `SignerV4::convertBodyToStream`
People with s3:1.0.0 and core: 1.0.1 will call `SignerV4::convertBodyToStream` which still works
People with s3:1.1.0 and core: 1.0.1 will call `SignerV4ForS3::convertBodyToStream`